std_msgs/Header header
geometry_msgs/Vector3 xyz
geometry_msgs/Quaternion q
geometry_msgs/Vector3 v
geometry_msgs/Vector3 w
geometry_msgs/Vector3 a
float32 dt
float32 isctrl
float32 isBlocking
geometry_msgs/Vector3 avoid
